k8s_spark/parquet.spark.helm/values.yaml

```diff
@@ -110,8 +120,19 @@ affinity: {}
 #   initialDelaySeconds: 5
 
 bitnami-spark:
+  enabled: true
   image:
     tag: 3.2.0-debian-10-r44
+  master:
```

Comment:
When I have `master` with no properties ( all are commented out), it complains. Once I comment it or add some properties with the same default values from https://github.com/bitnami/charts/tree/master/bitnami/spark, it works.

Reply:
I actually bumped into this this week with a different helm chart.  It appears to be a bug with the either bitnami Helm chart or Helm itself expecting there to be child keys/values where a parent exists.  If you omit the parent altogether ("master" in this case) or retain a comment, things appear to render normally.

Not sure there can be further action on this issue since it appears to be upstream.  Hopefully future versions of either Helm or Bitnami Spark will resolve this.

Reply:
I think we'll just comment it if there are variables are not updated for `master`. 
Thanks.